diff -aurN scribes-0.2.2.orig/setup.py scribes-0.2.2/setup.py
--- scribes-0.2.2.orig/setup.py	2005-12-01 07:10:39.000000000 +0100
+++ scribes-0.2.2/setup.py	2005-12-04 20:41:11.000000000 +0100
@@ -89,36 +89,6 @@
     	print "You need GNOME Python version 2.12 or later to run Scribes"
         exit(1)
 
-    try:
-
-        import gtksourceview
-
-    except ImportError:
-
-        print "You need the gtksourceview module in GNOME Python Extras version"
-        print " 2.12 or later to run Scribes."
-        exit(1)
-
-	try:
-
-		import gtkspell
-
-	except ImportError:
-
-		print "You need the gtkspell module in GNOME Python Extras version"
-        print " 2.12 or later to run Scribes."
-        exit(1)
-
-    # Check to see if Yelp is installed on the host system
-
-	cmd = "which yelp"
-	err, out = getstatusoutput(cmd)
-
-	if err:
-
-		print "You need Yelp version 2.12 or later to run Scribes."
-		exit(1)
-
 	# Check to see if GConf is installed on the host system
 
 	cmd = "which gconftool-2"
@@ -175,7 +145,7 @@
 	data_files=[
 		("share/applications", ["data/scribes.desktop"]),
 		("share/application-registry", ["data/scribes.application"]),
-		("share/pixmaps", ["data/scribes.svg"]),
+		("share/pixmaps", ["data/scribes.png"]),
 		("share/scribes/", ["data/scribes_toolbar_ui.xml"]),
 		("share/gnome/help/scribes/C", ["data/scribes.xml"]),
 		("share/gnome/help/scribes/C", ["data/legal.xml"]),
@@ -190,51 +160,3 @@
 	],
 	scripts=["scribes"],)
 
-
-################################################################################
-#
-#							GConf Installation
-#
-################################################################################
-
-# Get gconf's default source
-
-cmd = "gconftool-2 --get-default-source"
-err, out = getstatusoutput(cmd)
-
-# Set up the gconf environment variable.
-
-putenv('GCONF_CONFIG_SOURCE', out)
-
-# Install gconf to the default source
-
-cmd = "gconftool-2 --makefile-install-rule data/scribes.schemas"
-err, out = getstatusoutput(cmd)
-
-if out:
-
-    print "installing GConf schema files"
-
-if err:
-
-    print 'Error: installation of gconf schema files failed: %s' % out
-
-# Kill the GConf daemon
-
-cmd = "killall gconfd-2"
-err, out = getstatusoutput(cmd)
-
-if err:
-
-	print "Problem shutting down gconf."
-
-# Start the GConf daemon
-
-cmd = "gconftool-2 --spawn"
-err, out = getstatusoutput(cmd)
-
-if err:
-
-	print "Problem restarting down gconf."
-
-print "scribes installation complete"
