diff -aurN scribes-0.2.2.orig/data/scribes.desktop scribes-0.2.2/data/scribes.desktop
--- scribes-0.2.2.orig/data/scribes.desktop	2005-11-21 18:48:47.000000000 +0100
+++ scribes-0.2.2/data/scribes.desktop	2005-12-04 20:43:46.000000000 +0100
@@ -1,10 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Scribes Text Editor
+Name[pl]=Edytor tekstu Scribes
 Exec=scribes %f
 Terminal=false
 Type=Application
 StartupNotify=true
 MimeType=text/plain;
-Icon=scribes.svg
-Categories=Application;Utility;TextEditor;
\ Brak znaku nowej linii na koñcu pliku
+Icon=scribes
+Categories=GTK;GNOME;;Utility;TextEditor;
