--- scribes-0.2.4/setup.py	2006-02-03 06:00:10.000000000 +0100
+++ scribes-0.2.4/setup.py~	2006-02-03 12:50:42.484562750 +0100
@@ -126,7 +126,7 @@
 
 	return
 
-check_dependencies()
+#check_dependencies()
 
 # Trove classification
 
@@ -170,7 +170,7 @@
 	data_files=[
 		("share/applications", ["data/scribes.desktop"]),
 		("share/application-registry", ["data/scribes.application"]),
-		("share/pixmaps", ["data/scribes.svg"]),
+		("share/pixmaps", ["data/scribes.png"]),
 		("share/scribes/", ["data/scribes_toolbar_ui.xml"]),
 		("share/gnome/help/scribes/C", ["data/scribes.xml"]),
 		("share/gnome/help/scribes/C", ["data/legal.xml"]),
@@ -187,52 +187,5 @@
 	],
 	scripts=["scribes"],)
 
-
-################################################################################
-#
-#							GConf Installation
-#
-################################################################################
-
-# Get gconf's default source
-
-cmd = "gconftool-2 --get-default-source"
-err, out = getstatusoutput(cmd)
-
-# Set up the gconf environment variable.
-
-putenv('GCONF_CONFIG_SOURCE', out)
-
-# Install gconf to the default source
-
-cmd = "gconftool-2 --makefile-install-rule data/scribes.schemas"
-err, out = getstatusoutput(cmd)
-
-if out:
-
-	print "installing GConf schema files"
-
-if err:
-
-    print 'Error: installation of gconf schema files failed: %s' % out
-
-# Kill the GConf daemon
-
-cmd = "killall gconfd-2"
-err, out = getstatusoutput(cmd)
-
-if err:
-
-	print "Problem shutting down gconf."
-
-# Start the GConf daemon
-
-cmd = "gconftool-2 --spawn"
-err, out = getstatusoutput(cmd)
-
-if err:
-
-	print "Problem restarting down gconf."
-
 print "scribes installation complete"
 
